client/views/admin/info/InformationRoute.tsx

```diff
@@ -83,21 +90,22 @@ const InformationRoute = memo(function InformationRoute() {
 					</ButtonGroup>
 				</Page.Header>
 				<Page.ScrollableContentWithShadow>
-					<Callout type='danger'>
-						{t('Error_loading_pages')} {/* : {error.message || error.stack}*/}
-					</Callout>
+					<Callout type='danger'>{t('Error_loading_pages')}</Callout>
 				</Page.ScrollableContentWithShadow>
 			</Page>
 		);
 	}
 
+	if (isLoading) {
+		return <PageSkeleton />;
+	}
+
 	if (canViewStatistics) {
 		return (
-			<NewInformationPage
+			<InformationPage
 				canViewStatistics={canViewStatistics}
-				isLoading={isLoading}
 				info={info}
-				statistics={statistics}
+				statistics={statistics as IStats}
```

Comment:
What is the type of `statistics`? If it's `IStats | undefined`, we should narrow the type with a `if` clause instead of using type assertion.